src/main/java/opendcs/util/sql/ConnectionInTransaction.java

```diff
@@ -13,8 +13,18 @@ public ConnectionInTransaction(Connection realConnection)
         super(realConnection);
     }
 
+    /**
+     * So long as the actual state isn't changing, this can be a noop.
```

Comment:
```suggestion
     * Throws an Exception if the state would change, because we are in a transaction.
     * So long as the actual state isn't changing, this can be a noop.
```